Makefile

```diff
@@ -2,6 +2,8 @@ VERSION := $(shell echo $(shell git describe --tags 2>/dev/null || git log -1 --
 COMMIT := $(shell git rev-parse --short HEAD)
 DOCKER := $(shell which docker)
 PROJECTNAME=$(shell basename "$(PWD)")
+GOOS ?= linux
+GOARCH ?= amd64
```

Comment:
maybe these vars should be prefixed with `DOCKER`, otherwise its a little misleading why they are hardcoded to linux/amd64